src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -912,6 +935,8 @@ public virtual ToolComponent VisitToolComponent(ToolComponent node)
         {
             if (node != null)
             {
+                node.ShortDescription = VisitNullChecked(node.ShortDescription);
+                node.FullDescription = VisitNullChecked(node.FullDescription);
```

Comment:
same comment as above: this is strictly not part of this PR, but when i regenerated this file i noticed this additional check and thought of bringing is to attention in case it was mistakenly not included earlier?